baselines/Common Controls Minimum Viable Secure Configuration Baseline v0.1.md

```diff
@@ -1029,57 +939,20 @@ To add an app to the allowlist:
 
 3.  Click **Allowlisted Apps** to manage the allow list.
 
-## 13. Less Secure Apps
-
-This control disables legacy authentication and requires the use of modern authentication protocols based on federation for access from applications.
-
-Some older versions of common software may break when this control is implemented. Examples of these apps include:
-
--   Mails configured with POP3
--   Older versions of Outlook
-
-### Policies
-
-#### GWS.COMMONCONTROLS.13.1v0.1
-Access to Google Workspace applications by less secure apps that do not meet security standards for authentication SHALL be prevented.
-
-- Rationale
-  - You can block sign-in attempts from some apps or devices that are less secure. Apps that are less secure don't use modern security standards, such as OAuth. Using apps and devices that don't use modern security standards increases the risk of accounts being compromised. Blocking these apps and devices helps keep your users and data safe.
-- Last Modified: July 10, 2023
-
-- MITRE ATT&CK TTP Mapping
-  - [T1110: Brute Force](https://attack.mitre.org/techniques/T1110/)
-    - [T1110:001: Brute Force: Password Guessing](https://attack.mitre.org/techniques/T1110/001/)
-    - [T1110:002: Brute Force: Password Cracking](https://attack.mitre.org/techniques/T1110/002/)
-    - [T1110:003: Brute Force: Password Spraying](https://attack.mitre.org/techniques/T1110/003/)
-  - [T1566: Phishing](https://attack.mitre.org/techniques/T1566/)
-    - [T1566:002: Phishing: Spearphishing Link](https://attack.mitre.org/techniques/T1566/002/)
-
-### Resources
-
--   [GWS Admin Help \| Control access to less secure apps](https://support.google.com/a/answer/6260879?hl=en)
```

Comment:
Recommend also adding this as a resource to the "Authorized Google Marketplace Apps" group.

Reply:
@adhilto Added.